src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -41,7 +44,7 @@ const AnnotationActivityMenu = ({ canDelete, id, onDeleteConfirm }: AnnotationAc
 
     return (
         <TetherComponent {...tetherProps}>
-            {canDelete && (
```

Comment:
If the user has neither permission, do they seen an empty list?

Reply:
In `AnnotationActivity`, `isMenuVisible` will be set to false if user has neither permission which means that we should not see the ellipsis so we wouldn't get to an empty list state.